pkgs/development/tools/knightos/kpack/default.nix

```diff
@@ -12,9 +12,7 @@ stdenv.mkDerivation rec {
     sha256 = "1l6bm2j45946i80qgwhrixg9sckazwb5x4051s76d3mapq9bara8";
   };
 
-  nativeBuildInputs = [ cmake ];
-
-  buildInputs = [ asciidoc libxslt.bin docbook_xsl ];
+  nativeBuildInputs = [ cmake asciidoc libxslt.bin ];
```

Comment:
when you fix buildInputs vs nativeBuildInputs it probably deserves strictDeps = true, to avoid potential regressions.

Reply:
Done.